test/jsinspect_test.js

```diff
@@ -249,6 +249,31 @@ exports.jsinspect = {
           }
         );
       }
+    },
+
+    reportToFile: function(test) {
+      fs.mkdir('tmp', function () {
```

Comment:
Right now `fs.rmdir` wouldn't run if any of the assertions failed. I'd suggest placing it in /tmp or the os tmp dir :) https://github.com/raszi/node-tmp :)